src/main/java/com/uber/cadence/internal/worker/PollerOptions.java

```diff
@@ -60,6 +62,8 @@ public static PollerOptions getDefaultInstance() {
 
     private String pollThreadNamePrefix;
 
+    private Boolean pollOnlyIfHasCapacity = Boolean.FALSE;
```

Comment:
How about change to `pollOnlyIfExecutorHasCapacity` to be more precise?

Reply:
updated